doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,262 @@
+   :alt: Opening the Asset tracker application
+
+   Opening the Asset tracker application for the nrf9160_pca20035ns build target
+
+.. note::
+
+   The *Board Directory* folder can be found in the following location: ``ncs/nrf/boards/arm``.
+
+
+4. Click **OK** to import the project into SES. You can now work with the project in the IDE.
+
+   .. include:: gs_programming.rst
+      :start-after: build_SES_projimport_note_start
+      :end-before: build_SES_projimport_note_end
+
+#. To build the sample or application:
+
+   a. Select your project in the Project Explorer.
+   #. From the menu, select **Build -> Build Solution**.
+      This builds the project.
+
+   You can find the output of the build, which includes the merged HEX file containing both the application and the SPM, in the ``zephyr`` subfolder in the build directory.
+
+.. prog_extdebugprobe_start
+
+6. To program the sample or application:
+
+   a. Set the Thingy:91 SWD selection switch (**SW2**) to **nRF91** or **nRF52** depending on whether you want to program the nRF9160 SiP or the nRF52840 SoC component.
+   #. Connect the Thingy:91 to the debug out port on a 10-pin external debug probe, for example, nRF9160 DK (Development Kit), using a 10-pin JTAG cable.
+
+      .. note::
+
+         If you are using nRF9160 DK as the debug probe, make sure that **VDD_IO (SW11)** is set to 1.8 V on the nRF9160 DK.
+
+   #. Connect the external debug probe to the PC using a USB cable.
+   #. Make sure that the Thingy:91 and the external debug probe are powered on.
+
+.. prog_extdebugprobe_end
+..
+
+   e. In SES, select **Target -> Connect J-Link**.
+   #. Select **Target -> Download zephyr/merged.hex** to program the sample or application onto Thingy:91.
+   #. The device will reset and run the programmed sample or application.
+
+.. _build_pgm_cmdline:
+
+Building and programming on the command line
+============================================
+
+Complete the command line build setup (see the instructions for :ref:`Windows<build_environment_cli_win>`, :ref:`Linux<build_environment_cli_linux>`, and :ref:`macOS<build_environment_cli_mac>`) before you start building |NCS| projects on the command line.
+
+To build and program the source code from the command line, complete the following steps:
+
+1. Open a terminal window.
+#. Go to the specific sample or application directory.
+   For example, the folder path is ``ncs/nrf/applications/asset_tracker`` when building the source code for the :ref:`asset_tracker` application on the nRF9160 SiP component and ``ncs/nrf/samples/usb/usb_uart_bridge`` when building the source code for the :ref:`usb_uart_bridge_sample` sample on the nRF52840 SoC component.
+
+#. Make sure that you have the required version of |NCS| repository by pulling the |NCS| repository, `fw-nrfconnect-nrf`_ on GitHub using the procedures described in :ref:`dm-wf-get-ncs` and :ref:`dm-wf-update-ncs`.
+
+#. To get the rest of the dependencies, run the ``west`` command as follows:
```

Comment:
```suggestion
#. To get the rest of the dependencies, run the ``west update`` command as follows:
```

Either this or just "``west`` command" in the following step - both works, it should just be consistent. :)

Reply:
Modified